CMSIS/NN/Tests/UnitTest/TestCases/test_arm_convolve_s8/test_arm_convolve_s8.c

```diff
@@ -1256,3 +1257,87 @@ void conv_dilation_golden_arm_convolve_s8(void)
     TEST_ASSERT_EQUAL(expected, result);
     TEST_ASSERT_TRUE(validate(output, output_ref, output_ref_size));
 }
+
+void conv_5_arm_convolve_s8(void)
+{
+    const arm_cmsis_nn_status expected = ARM_CMSIS_NN_SUCCESS;
+    q7_t output[CONV_5_DST_SIZE] = {0};
+
+    cmsis_nn_context ctx;
+    cmsis_nn_conv_params conv_params;
+    cmsis_nn_per_channel_quant_params quant_params;
+    cmsis_nn_dims input_dims;
+    cmsis_nn_dims filter_dims;
+    cmsis_nn_dims bias_dims;
+    cmsis_nn_dims output_dims;
+
+    const q31_t *bias_data = conv_5_biases;
+    const q7_t *kernel_data = conv_5_weights;
+    const q7_t *input_data = conv_5_input;
+    const q7_t *output_ref = conv_5_output_ref;
+    const int32_t output_ref_size = CONV_5_DST_SIZE;
+
+    input_dims.n = CONV_5_INPUT_BATCHES;
+    input_dims.w = CONV_5_INPUT_W;
+    input_dims.h = CONV_5_INPUT_H;
+    input_dims.c = CONV_5_IN_CH;
+    filter_dims.w = CONV_5_FILTER_X;
+    filter_dims.h = CONV_5_FILTER_Y;
+    output_dims.w = CONV_5_OUTPUT_W;
+    output_dims.h = CONV_5_OUTPUT_H;
+    output_dims.c = CONV_5_OUT_CH;
+
+    conv_params.padding.w = CONV_5_PAD_X;
+    conv_params.padding.h = CONV_5_PAD_Y;
+    conv_params.stride.w = CONV_5_STRIDE_X;
+    conv_params.stride.h = CONV_5_STRIDE_Y;
+    conv_params.dilation.w = CONV_5_DILATION_X;
+    conv_params.dilation.h = CONV_5_DILATION_Y;
+
+    conv_params.input_offset = CONV_5_INPUT_OFFSET;
+    conv_params.output_offset = CONV_5_OUTPUT_OFFSET;
+    conv_params.activation.min = CONV_5_OUT_ACTIVATION_MIN;
+    conv_params.activation.max = CONV_5_OUT_ACTIVATION_MAX;
+    quant_params.multiplier = (int32_t *)conv_5_output_mult;
+    quant_params.shift = (int32_t *)conv_5_output_shift;
+
+    int32_t buf_size = arm_convolve_s8_get_buffer_size(&input_dims, &filter_dims);
+    ctx.buf = malloc(buf_size);
+    ctx.size = 0;
+
+    arm_cmsis_nn_status result = arm_convolve_s8(&ctx,
```

Comment:
It would be better to use the dedicated API arm_convolve_1_x_n_s8() here instead of arm_convolve_s8

Reply:
This unit test is not targeted for arm_convolve_1_x_n_s8. It will fail with arm_convolve_1_x_n_s8 for example without the wrapper updated.

Reply:
right,..it is test_arm_convolve_s8.c.. Thanks for the clarification..